src/containers/search/SearchForm.tsx

```diff
@@ -254,7 +262,44 @@ export const SearchForm: React.FC = () => {
   return (
     <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
       <SearchPopovers {...popOversState} />
-      <Stack direction="column">
+      <Box
+        sx={
+          !open && isMobile && closed
+            ? { background: 'white', width: '100%', p: 2 }
```

Comment:
i believe `white` should not be hardcoded

Reply:
fixed